draft-normington-jsonpath-latest.xml

```diff
@@ -63,6 +63,20 @@
         <email>esurov.tsp@gmail.com</email>
      </address>
     </author>
+    <author fullname="Marko Mikulicic" initials="M" role="editor" surname="Mikulicic">
```

Comment:
Yes, according to the [definition](https://tools.ietf.org/html/rfc7991#section-2.7.6) of the author tag, `editor` is the only option for a role. f you'd prefer not to act an an editor:
```suggestion
    <author fullname="Marko Mikulicic" initials="M" surname="Mikulicic">
```

Reply:
thanks